.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -11,6 +11,8 @@ env:
   SLACK_ICON: https://pbs.twimg.com/profile_images/1274014582265298945/OjBKP9kn_400x400.png
   SLACK_USERNAME: ${{ secrets.BOT_USERNAME }}
   SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  MSG_MINIMAL: event,action url,commit
```

Comment:
what is the MSG_MINIMAL ?

Reply:
The message format displayed on Slack. If you add `ref` to the list you'll find the current notification format.
Removing `ref` allows to get rid of the default behavior of the action.

From action's README:

> If set to true, removes: Ref, Event, Actions URL and Commit from the message. You can optionally whitelist any of these 4 removed values by passing it comma separated to the variable instead of true. (ex: MSG_MINIMAL: event or MSG_MINIMAL: ref,actions url, etc.)